examples/meta/src/meta_api/kwargs.sg

```diff
@@ -22,17 +22,14 @@
 # This also fails (keywords not allowed outside initialisation variables)
 #kernel_factory("GaussianKernel", a=glob_fun(ordinary_argument))
 
+# doesnt work in ruby (TODO, don't allow in meta grammar)
```

Comment:
@sorig we will need to block this, ruby doesn't like it

Reply:
What does ruby say? I can disable nested global calls, but that may disable some nested calls that are actually useful.

Reply:
I think we're just using the wrong syntax. [See here](https://en.wikibooks.org/wiki/Ruby_Programming/Syntax/Method_Calls#Method_Calls)